frontend/src/Dashboard.jsx

```diff
@@ -25,6 +27,11 @@ const Dashboard = (props) => {
   } = props;
   const classes = useStyles();
 
+  useEffect(() => {
+    socket.init();
+    socket.subscribe('ping', (data) => console.log(data));
```

Comment:
Ik weet niet 100% zeker hoe effectief dit is, maar is het misschien een idee om dit te vervangen met `info.log(data)`?